rpc/shared/models.go

```diff
@@ -14,3 +14,10 @@ type Example struct {
 	Data  string `json:"data"`
 }
 
+type Information struct{
```

Comment:
Information is not a good name I believe for this struct.

Reply:
Check it out now? "APIReference"